epilogue_1_how_to_get_there_from_here.asciidoc

```diff
@@ -485,9 +534,14 @@ This is a part 2 thing really, but basically, don't sally forth and implement
 your own event-driven microservices architecture without reading lots, lots
 more on the subject.
 
+
 https://martinfowler.com/books/eip.html[Enterprise Integration Patterns] by
 (as always) Martin Fowler is a pretty good start.
 
+// (EJ2) It would be worth saying something about NoSql here (or in "Questions our Tech Reviwers..?"). 
```

Comment:
Gladly! Where does it fit? 

I'll happily include the data intensive link in the footguns section.

Reply:
I think it fits in topically either with Repositories, or the conversation about data models vs. domain models. The footgun here is this: If you attempt to apply relational modeling techniques to a non-relational store, you are at risk for ending up with concurrency issues or lost data.  DDD patterns like aggregates, entities, and value objects are useful for designing a data model appropriate for NoSql store.